tools/reviewer/web/src/app/dashboard/reviews-panel/review/services/review.service.ts

```diff
@@ -1,7 +1,6 @@
 import { Injectable } from '@angular/core';
 
-import { Diff, Reviewer } from '@/shared';
-import { AuthService } from './auth.service';
+import { Diff, Reviewer, AuthService } from '@/shared';
```

Comment:
I think it'd be more clear, if we import proto and services separately. And to specify type by the path.
```typescript
import { Diff, Reviewer } from '@/shared/shell';
import { AuthService } from '@/shared/services';
```
WDYT?